etiquette.md

```diff
@@ -16,4 +18,11 @@ There will be one exception to this rule: Meeting summaries published to this re
 
 6. Mind your microphone, and be focused on the discussion.  If you are not devoting your attention to the meeting, you should instead read the meeting summary.
 
-7. Chairpeople should be dispassionate toward the discussion topic, and not partipate in the discussion.
\ No newline at end of file
+7. Chairpeople should be dispassionate toward the discussion topic, and not participate in the discussion.
+
+8. Refrain from posting to social media before Meeting summary has been released tohelp drive effective discussion rather than speculation, if there is an urgency to begin public discussion, feel welcome to collaborate on that summary first.
```

Comment:
tohelp -> to help (add space)
speculation, if -> speculation. If (new sentence)